kotsadm/pkg/apiserver/bootstrap.go

```diff
@@ -106,6 +102,25 @@ func bootstrapIdentity() error {
 		}
 		identityConfig := obj.(*kotsv1beta1.IdentityConfig)
 
+		installationFile := filepath.Join(currentArchivePath, "upstream", "userdata", "installation.yaml")
+		installationData, err := ioutil.ReadFile(installationFile)
```

Comment:
we should check if this does not exist

Reply:
this should always exist.  if it doesn't we can't proceed.